### PR TITLE
Update @altinn/altinn-components to v0.10.0 and refactor related components

### PR DESCRIPTION
**Pull request by sonwit**
Update @altinn/altinn-components to v0.10.0 and refactor related components

## Description
Update @altinn/altinn-components to v0.10.0
Use AccessPackageList and AccessAreaList from altinn-components

## Related Issue(s)
- N/A

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/features/amUI/userRightsPage/AccessPackageSection/DelegatedPackagesList.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
-import React from 'react';
-import { ListItem } from '@altinn/altinn-components';
 import { Paragraph } from '@digdir/designsystemet-react';
 import { useTranslation } from 'react-i18next';
+import { Button } from '@altinn/altinn-components';
 
 import type { AccessPackage, AccessPackageDelegation } from '@/rtk/features/accessPackageApi';
-import { List } from '@/components';
+
+import { AccessPackageList } from '../../../../../../altinn-components/lib/components';
```
Comment (allinox) [inline src/features/amUI/userRightsPage/AccessPackageSection/DelegatedPackagesList.tsx:7]: Another local inclusion?

File: src/features/amUI/userRightsPage/SingleRightsSection/SingleRightItem.tsx [hunk 2]
```diff
@@ -2,11 +2,12 @@
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 
 import React, { useRef } from 'react';
-import { ListItem } from '@altinn/altinn-components';
 
+//import { ResourceListItem } from '@altinn/altinn-components';
 import { type ServiceResource } from '@/rtk/features/singleRights/singleRightsApi';
 import { type Party } from '@/rtk/features/lookupApi';
 
+import { ResourceListItem } from '../../../../../../altinn-components/lib/components';
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/amUI/userRightsPage/SingleRightsSection/SingleRightItem.tsx:10]: 🤦‍♂️

File: src/features/amUI/userRightsPage/AccessPackageSection/DelegatedPackagesList.tsx (hunk 1 above)
Comment (sonwit) [reply src/features/amUI/userRightsPage/AccessPackageSection/DelegatedPackagesList.tsx:7]: 🤦‍♂️

**Review by allinox — APPROVED** [review]
🙌

**Pull request merged**